api/src/db/knexfile.js

```diff
@@ -31,12 +31,12 @@ const def = {
 
 module.exports = {
   test: test,
-  development: Object.assign({}, {
+  development: Object.assign(def, {
     seeds: {
       directory: path.join(__dirname, 'seeds', 'development')
     },
     debug: process.env.KNEX_DEBUG || false
-  }, def),
+  }),
   staging: def,
```

Comment:
@sethvincent @scisco we don't need a staging def anymore, right? We can remove it in this PR as well. 
```suggestion
```